wp-job-manager-template.php

```diff
@@ -548,13 +548,13 @@ function wpjm_get_the_job_description( $post = null ) {
 		return null;
 	}
 
-	$description = apply_filters( 'the_job_description', get_the_content( $post ) );
+	$description = apply_filters( 'the_job_description', wp_kses_post( $post->post_content ) );
```

Comment:
Seems like when we're calling `wpjm_the_job_description`, we'll be running `kses` twice. Is this necessary? Could we just call `WP_Job_Manager_Post_Types::output_kses_post` here, and not worry about it in `wpjm_the_job_description`?

Could be that there's a good reason to call it twice, and I'm just not seeing it 😄

Reply:
This comes from above comment. The issue is we don’t want to allow `iframes` from user input _but_ we do want to allow them when they come from embeds (from within `the_job_description` hooks). We really probably don’t need to filter it again (post embed), but seems to be standard to it on output as well (to catch other things from hooks?).

One other note about this: there is a tiny bit of functionality change happening here. `get_the_content` was being called incorrectly and I would have had to override the global `$post` variable to do it correctly. With that, pagination inside of a single post is removed. I don’t know if it would have really worked before and doubt it was in-use, but just thought it best to note it.

Reply:
Works for me! I don't think the overhead of calling it twice will be significant, just wanted to be clear.

Yeah, I see what you mean about `get_the_content`. Definitely was wrong before. Hopefully the change doesn't cause weird side-effects, but it seems like we should be ok.